src/components/menu/Menu.component.tsx

```diff
@@ -1,11 +1,65 @@
-import React from 'react';
+import React from "react";
+import img1 from './images/menu1.png'; 
+import header from './images/header.png';
+import img2 from './images/menu2.png';
+import style from './Menu.module.css';
 import Footer from '../footer/Footer';
 
-export default function Menu() {
+
+ function Menu() {
+   const image1 = "https://www.notion.so/janac/Menu-Page-dishes-c97e397570384b69a4ce291d02dde758#88140f350d3e40d5986c2a7791ae3709";
```

Comment:
Use variable names that describe the content, so change `image1` to `menuImageUrl`